codeserver/rhel9-python-3.9/Dockerfile

```diff
@@ -0,0 +1,90 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG CODESERVER_VERSION=v4.16.1
+
+LABEL name="odh-notebook-code-server-c9s-python-3.9" \
+      summary="Code Server (VS Code) image with python 3.9 based on CentOS Stream 9" \
+      description="Code Server (VS Code) image with python 3.9 based on CentOS Stream 9" \
+      io.k9s.display-name="Code Server (VS Code) image with python 3.9 based on CentOS Stream 9" \
+      io.k9s.description="Code Server (VS Code) image with python 3.9 based on CentOS Stream 9" \
+      authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
+      io.openshift.build.commit.ref="main" \
+      io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/codeserver/c9s-python-3.9" \
+      io.openshift.build.image="quay.io/opendatahub/workbench-images:codeserver-c9s-python-3.9"
```

Comment:
Wrong links on these two lines.

Also, at the moment we point in downstream to the upstream locations... this is something that we should update in the future probably as a bulk update - eventually utilizing something from opendatahub-io/notebooks#309.